_src/Evasions/techniques/AVBypass.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "Evasions: Bypassing Security Solutions"
+title-image: "/assets/icons/AVBypass.svg"
+categories: evasions
+tags: AVBypass
```

Comment:
Please use lower case score for MD namings, as they all come in lower case except for the recent addition